lib/cli.js

```diff
@@ -142,7 +150,7 @@ exports.run = () => {
 		resolveTestsFrom: cli.input.length === 0 ? projectDir : process.cwd(),
 		projectDir,
 		timeout: conf.timeout,
-		concurrency: conf.concurrency ? parseInt(conf.concurrency, 10) : 0,
+		concurrency: conf.concurrency ? parseInt(conf.concurrency, 10) : 1,
```

Comment:
The default should remain `0`. This allows AVA to pick an optimal value for your machine, see https://github.com/avajs/ava/blob/465fcecc9ae0d3274d4d41d3baaca241d6a40130/api.js#L164:L167.